package.json

```diff
@@ -97,6 +97,7 @@
 		"/retire-js",
 		"/oclif.lock"
 	],
+	"types": "/src/types.d.ts",
```

Comment:
I'm not sure I agree with us using *.d.ts files for this purpose because it adds these types globally instead of requiring imports. Here is a nice video on the subject: https://www.youtube.com/watch?v=zu-EgnbmcLY

Thoughts?

Reply:
Thank you @stephen-carter-at-sf. That makes perfect sense. Let me make this change and send an update.